### ISSUE TITLE
[FOCUS MANAGEMENT]: Focus SHOULD move to the next question after selecting Yes or No

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#9967]
**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

![Screenshot from 2020-06-08 16-47-23](https://user-images.githubusercontent.com/57469/84079956-68245480-a9a9-11ea-8428-a9145a7645bd.png)

In the screenshot above, after selecting the previous yes/no, I would expect focus to move to the question  — and not have a yellow outline. The platform guidance is to not have the focus outline on text that is in focus, only on interactive items.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screen reader user, I want to move to the next question after making a selection for the current question.

## Environment

* Operating System: all, iOS, Android
* Browser: Safari, Chrome or Android native browser
* Screenreading device: VoiceOver, TalkBack
* Server destination: staging? Mike Chelen's local server?

## Steps to Recreate

1. Enter `<IP ADDRESS>` in browser
2. Start screenreading device listed in Environment
3. Navigate through the form, selecting Yes or No for each question
4. Verify where the focus moves

**Comment by stephenBarrs** [comment]
This issue is confirmed to occur on Android devices with #9608

**Comment by joshkimux** [comment]
@batemapf was this ticket addressed in #9608(https://github.com/department-of-veterans-affairs/va.gov-team/issues/9608)? If so, could we close it out? Otherwise, who could we assign this too? 🙇

**Comment by artsymartha68** [comment]
I was curious so I looked up the covid screener usage. 18K people used it in March 2023. That said, I'm pretty sure we don't have any folks available to work on this any more. 

<img width="1182" alt="Screen Shot 2023-03-29 at 9 55 31 AM" src="https://user-images.githubusercontent.com/47792423/228561229-19adddcb-09ea-42a0-9772-c9691997bb11.png">

**Comment by joshkimux** [comment]
@artsymartha68 thank you for this! Let me leave a note on our dashboard that this doesn't have a team

**Comment by joshkimux** [comment]
@artsymartha68 one more question-- do you have any recommendations on how we should move forward with this one?
